docs/index.bs

```diff
@@ -1744,12 +1745,36 @@ spec: webappsec-referrer-policy; urlPrefix: https://w3c.github.io/webappsec-refe
 </section>
 
 <section>
-  <h2 id="link-type-serviceworker">Link type "<code>serviceworker</code>"</h2>
+  <h2 id="link-type-serviceworker">Link type "`serviceworker`"</h2>
 
-  The <dfn id="dfn-link-type-serviceworker" lt="serviceworker link type"><code>serviceworker</code></dfn> keyword may be used with <{link}> elements. This keyword creates an <a>external resource link</a> (<dfn id="dfn-serviceworker-link">serviceworker link</dfn>) that is used to declare a [=/service worker registration=] and its [=service worker registration/scope url=].
+  A [=/service worker registration=] and its [=service worker registration/scope url=] are created by a <dfn id="dfn-serviceworker-link">serviceworker link</dfn>, which is declared using a <a href="#serviceworker-link-header">"serviceworker" `Link` header</a> or a <{link}> element whose <{link/rel}> attribute contains the keyword "<{link/rel/serviceworker}>".
 
   <section>
-    <h3 id="link-header-processing">Processing the <code>Link</code> header</h3>
+    <h3 id="serviceworker-link-header">Declaring a "serviceworker" <code>Link</code> header</h3>
+
+    A <a>serviceworker link</a> can be declared using a `Link` header</a> [[!RFC5988]] with "`serviceworker`" as the value of the "`rel`" parameter and a [=service worker/script url=] as the <a>target IRI</a>, and the following optional <a>target attributes</a>:
+
+    : `scope`
+    :: Value: A [=service worker registration/scope url=].
```

Comment:
I think it's better not to reference obsoleted concepts like URI. Instead I'd do ", serialized", linking to https://url.spec.whatwg.org/#concept-url-serializer (so I think that's `[=URL serializer|serialized=]`).